API/Requests/Reservation/GetEmployeeReservationTimesRequest.cs

```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Requests.Reservation
+{
+    public class GetEmployeeReservationTimesRequest
+    {
+        [FromRoute(Name = "employeeId")]
+        public Guid EmployeeId { get; set; }

```

Comment:
Honestly, jei čia Guid, gali tiesiog dėt į route'ą Guid, nebent planuose yra plėst šitą request'ą ateityje

Reply:
Kai kurie kiti failai irgi taip padaryti, gal taip ir paliekam